reward/staking_info_cache.go

```diff
@@ -64,4 +64,5 @@ func (sc *stakingInfoCache) add(stakingInfo *StakingInfo) {
 		}
 	}
 	sc.cells[stakingInfo.BlockNum] = stakingInfo
+	logger.Debug("Add a new stakingInfo to stakingInfoCache", "stakingInfo", stakingInfo)
```

Comment:
There is already info log after this. how about remove this?

```
logger.Info("Add a new stakingInfo to the stakingInfoCache", "stakingInfo", stakingInfo)
```

Reply:
Cache is not updated if it already has the value in cache. The two logs differ from each other.

Reply:
Got it. But I think logging `stakingInfo` is useless. How about this?
```suggestion
	logger.Debug("Add a new stakingInfo to stakingInfoCache")
```

Reply:
I took your suggestion below. It is now 
```
	logger.Info("Add a new stakingInfo to stakingInfoCache and stakingInfoDB", "blockNum", blockNum)
	logger.Debug("Added stakingInfo", "stakingInfo", stakingInfo)
```

Reply:
Thanks. But could you remove `stakingInfo` heare also like above comment?
`stakingInfo` log is duplicated.

Reply:
Yes, it now prints block number instead of stakingInfo